ui/component/fileTitle/view.jsx

```diff
@@ -39,14 +45,32 @@ function FileTitle(props: Props) {
         </React.Fragment>
       }
       actions={
-        <div>
-          <div className="section">
-            <FileActions uri={uri} />
+        isNsfwBlocked ? (
+          <div className="main--empty">
+            <h2 className="card__title card__title">
```

Comment:
Since they are in the same card, this shouldn't use `card__title`. I think the generic `h2` will work.